content/tutorials/manage-the-rippled-server/troubleshooting/server-wont-start.md

```diff
@@ -0,0 +1,169 @@
+# rippled Server Won't Start
+
+This page explains possible reasons the `rippled` server does not start successfully, and how to fix them.
```

Comment:
```
< This page explains possible reasons the `rippled` server does not start successfully, and how
---
> This page explains possible reasons the `rippled` server does not start and how
```

Reply:
I mean, debatably, the server usually _starts_ but doesn't always get through the entire startup successfully. ;) But point taken.